File: Data Structures & Algorithms/max-consecutive-ones/submission-0.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int max = 0;
        int cur = 0;
        for (int i = 0; i < nums.size(); i++) {
            if (nums[i] == 1) {
                cur++;
            }
            if (nums[i] == 0){
                if (cur >= max){
                max = cur;
                }
                cur = 0;
            }
        }
        
        max = std::max(max, cur);
        return max;
    }
};

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:   
        maxlength = 0
        length = 0

        for i in nums:
            if i == 1: 
                length += 1 
            else: 
                length = 0
            maxlength = max(length, maxlength) 
        return maxlength
            

        